src/basicElements/args.ts

```diff
@@ -133,7 +133,7 @@ export class Args {
    * @param offset - The optional offset to start deserializing from.
    */
   constructor(
-    public serialized = new Uint8Array(),
+    public serialized: Uint8Array = new Uint8Array(),
```

Comment:
fix: Type 'Uint8Array<ArrayBufferLike>' is not assignable to type 'Uint8Array<ArrayBuffer>
Might not be the best way to fix it